### PR TITLE
fix(Roaring): Remove pack from posting list

### PR DESCRIPTION
**Pull request by ahsanbarkati**
We no longer user group varint based UidPack in posting lists. This PR removes it from the proto message and also removes the corresponding stale codes.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7535)

### CODE REVIEW

File: worker/restore.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func loadFromBackup(db *badger.DB, in *loadBackupInput) (uint64, uint64, error)
 					return 0, 0, errors.Wrapf(err, "while reading backup posting list")
 				}
 				pl := posting.FromBackupPostingList(backupPl)
-				shouldSplit := pl.Size() >= (1<<20)/2 && len(pl.Pack.Blocks) > 1
+				shouldSplit := pl.Size() >= (1<<20)/2
```
Comment (jarifibrahim) [inline worker/restore.go:197]: should we check the len(bitmap) over here? Similar to how we were checking len(pl.Pack.Blocks)?

Comment (ahsanbarkati) [reply worker/restore.go:197]: It is not strictly required. We call rollups if `shouldSplit` is `true`, which will then be a noop as it checks for the cardinality. 
Plus, checking for cardinality of a bitmap is expensive (we will have to de-serialize the bitmap). So I think that the check on posting list size can act as a preliminary filter here.

Comment (ahsanbarkati) [reply worker/restore.go]: Done

Comment (NamanJain8) [reply worker/restore.go]: Also add this check in bulk loader.

**Review by NamanJain8 — APPROVED** [review]

**Pull request merged**